README.MD

```diff
@@ -12,4 +12,50 @@ This mod doesn't do anything by itself! You have to create a custom item [like t
 
 ## How to create your own custom item
 
-Follow the instructions [here](https://git.tostiman.com/tostiman/dv_blahaj).
\ No newline at end of file
+Follow the instructions [here](https://git.tostiman.com/tostiman/dv_blahaj).

```

Comment:
Thanks for taking the time to write this!  
But now we kinda have 2 guides: this page and the README of Blahaj. A single, step-by-step instructions page would be better. Would you be willing to merge the 2 into a single guide here?

Reply:
We could still link to Blahaj and tell users they can use that as a template.

Reply:
Yeah, I'm not opposed to that, but ultimately they are two different parts of the same whole.  What you have on the blahaj is more a tutorial, and also covers something that this mod doesn't currently which is how to actually export your asset bundle.

Perhaps keeping some of the minimum requirements listed here, but then set up a wiki with tutorial, item guide, gadget guide, etc?